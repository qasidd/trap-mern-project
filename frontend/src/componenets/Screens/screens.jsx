import { Card,CardBody,CardTitle,CardFooter,CardText,CardImg } from 'reactstrap';
import Standardseatingplan from '../../resources/Standardscreenseatingplan.png'
import Standarddecor from '../../resources/Standard screen decor.jpg'
import Deluxeseatingplan from '../../resources/deluxe screen seating plan.png'
import Deluxedecor from '../../resources/deluxe screen decor.jpg'
const Screens = () => {
    return (
        
        <div>
            <Card>
                <CardImg variant="top" src={Standardseatingplan} width="500" length="600" />
                <CardBody>
                    <div className="container text-center">
                        <CardTitle>Seating plan for Standard screen</CardTitle>
                    </div>
                    <CardText>
                        Here you can see a layout of our Standard viewing Screens
                    </CardText>
                </CardBody>
                <CardFooter>
                    <div className=" container text-center">
                        <small className="text-muted">Last updated 6 months ago</small>
                    </div>
                </CardFooter>
            </Card>

            <Card>
                <CardImg variant="top" src={Standarddecor} width="500" length="600"/>
                <CardBody>
                    <div className="container text-center">
                        <CardTitle>Interior of our Standard Screens</CardTitle>
                    </div>
                    <CardText>Here you can see the interior of our Standard viewing screens. {' '}
                    </CardText>
                    
                </CardBody>
                <CardFooter>
                    <div className=" container text-center">
                        <small className="text-muted">Last updated 1 month ago</small>
                    </div>
                </CardFooter>
            </Card>

            <Card>
                <CardImg variant="top" src={Deluxeseatingplan}width="500" length="600" />
                <CardBody>
                    <div className="container text-center">
                        <CardTitle>The Seating plan for our Deluxe Directors Box seats</CardTitle>
                    </div>
                    <CardText> Here you can see the seating plan for our premier Directors Box Seats
                    </CardText>
                </CardBody>
                <CardFooter>
                    <div className=" container text-center">
                        <small className="text-muted">Last updated 27/08/2020</small>
                    </div>
                </CardFooter>
            </Card>
            <Card>
                <CardImg variant="top" src={Deluxedecor} width="500" length="600"/>
                <CardBody>
                    <div className="container text-center">
                        <CardTitle>The Interior for our Deluxe Directors Box seats</CardTitle>
                    </div>
                    <CardText> Here you can see the Layout and decor of our premier Directors Box Seats
                    </CardText>
                </CardBody>
                <CardFooter>
                    <div className=" container text-center">
                        <small className="text-muted">Last updated 27/08/2020</small>
                    </div>
                </CardFooter>
            </Card>
      
        </div>
    )


}

export default Screens;