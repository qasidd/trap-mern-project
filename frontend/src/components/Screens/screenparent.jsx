import Screens from "./screens"

const Screenparent = () => {
    return (
        <div className="container mainContent">
            <h1 className="display-4">Screens</h1>
            <br></br>
            <Screens />
        </div>
    );
}

export default Screenparent