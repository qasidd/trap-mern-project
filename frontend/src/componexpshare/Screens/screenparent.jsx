import Screens from "./screens"

const Screenparent = () => {
    return (
        <>
        <section>
            <div className="container text-center">
            <h2>Screens</h2>
            </div>
        </section>
        <br></br>
        <Screens/>
        </>
    );
}

export default Screenparent