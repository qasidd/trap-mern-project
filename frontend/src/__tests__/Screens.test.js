import Screens from '../components/Screens/screenparent';
import '@testing-library/jest-dom';
import { expect, it } from '@jest/globals';
import { create } from 'react-test-renderer';

it(`It should match the snapshot every time it's rendered`, () => {
    const instance = create(<Screens/>).toJSON();
    expect(instance).toMatchSnapshot();
    
})