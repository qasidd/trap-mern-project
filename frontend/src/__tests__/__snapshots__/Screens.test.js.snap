// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the snapshot every time it's rendered 1`] = `
Array [
  <section>
    <div
      className="container text-center"
    >
      <h2>
        Screens
      </h2>
    </div>
  </section>,
  <br />,
  <div>
    <div
      className="card"
    >
      <img
        className="card-img"
        length="600"
        src="Standardscreenseatingplan.png"
        variant="top"
        width="500"
      />
      <div
        className="card-body"
      >
        <div
          className="container text-center"
        >
          <div
            className="card-title"
          >
            Seating plan for Standard screen
          </div>
        </div>
        <p
          className="card-text"
        >
          Here you can see a layout of our Standard viewing Screens
        </p>
      </div>
      <div
        className="card-footer"
      >
        <div
          className=" container text-center"
        >
          <small
            className="text-muted"
          >
            Last updated 6 months ago
          </small>
        </div>
      </div>
    </div>
    <div
      className="card"
    >
      <img
        className="card-img"
        length="600"
        src="Standard screen decor.jpg"
        variant="top"
        width="500"
      />
      <div
        className="card-body"
      >
        <div
          className="container text-center"
        >
          <div
            className="card-title"
          >
            Interior of our Standard Screens
          </div>
        </div>
        <p
          className="card-text"
        >
          Here you can see the interior of our Standard viewing screens. 
           
        </p>
      </div>
      <div
        className="card-footer"
      >
        <div
          className=" container text-center"
        >
          <small
            className="text-muted"
          >
            Last updated 1 month ago
          </small>
        </div>
      </div>
    </div>
    <div
      className="card"
    >
      <img
        className="card-img"
        length="600"
        src="deluxe screen seating plan.png"
        variant="top"
        width="500"
      />
      <div
        className="card-body"
      >
        <div
          className="container text-center"
        >
          <div
            className="card-title"
          >
            The Seating plan for our Deluxe Directors Box seats
          </div>
        </div>
        <p
          className="card-text"
        >
           Here you can see the seating plan for our premier Directors Box Seats
        </p>
      </div>
      <div
        className="card-footer"
      >
        <div
          className=" container text-center"
        >
          <small
            className="text-muted"
          >
            Last updated 27/08/2020
          </small>
        </div>
      </div>
    </div>
    <div
      className="card"
    >
      <img
        className="card-img"
        length="600"
        src="deluxe screen decor.jpg"
        variant="top"
        width="500"
      />
      <div
        className="card-body"
      >
        <div
          className="container text-center"
        >
          <div
            className="card-title"
          >
            The Interior for our Deluxe Directors Box seats
          </div>
        </div>
        <p
          className="card-text"
        >
           Here you can see the Layout and decor of our premier Directors Box Seats
        </p>
      </div>
      <div
        className="card-footer"
      >
        <div
          className=" container text-center"
        >
          <small
            className="text-muted"
          >
            Last updated 27/08/2020
          </small>
        </div>
      </div>
    </div>
  </div>,
]
`;
